<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>控制模式选择</string>
   </property>
   <widget class="QComboBox" name="ctrl_mode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>89</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>指令模式</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>遥控模式</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>451</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>线速度控制</string>
   </property>
   <widget class="QSlider" name="linespeed_spit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="line_speed_sub1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="line_speed_sub10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QPushButton" name="line_speed_add10">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>++</string>
    </property>
   </widget>
   <widget class="QPushButton" name="line_speed_add1">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_speed_value">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>90</y>
     <width>451</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>角速度控制</string>
   </property>
   <widget class="QSlider" name="anglespeed_spit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="angle_speed_add10">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>++</string>
    </property>
   </widget>
   <widget class="QPushButton" name="angle_speed_sub10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="angle_speed_value">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="angle_speed_add1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="angle_speed_sub1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>931</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>故障清除指令</string>
   </property>
   <widget class="QPushButton" name="clean_battery_low_vol_error">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>清除电池欠压故障</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clean_battery_over_vol_error">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>清除电池过压故障</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clean_motot1_commu_error">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>清除电机1通讯故障</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clean_motot2_commu_error">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>清除电机2通讯故障</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clean_motot3_commu_error">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>清除电机3通讯故障</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clean_motot4_commu_error">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>清除电机4通讯故障</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clean_motor_over_temp_error">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>60</y>
      <width>181</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>清除电机驱动过温故障</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clean_motor_over_curr_error">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>30</y>
      <width>181</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>清除电机过流故障</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>941</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>系统状态反馈</string>
   </property>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>当前车状态</string>
    </property>
    <widget class="QLineEdit" name="car_status_feedback">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>当前控制模式</string>
    </property>
    <widget class="QLineEdit" name="ctrl_mode_feedback">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>电压值</string>
    </property>
    <widget class="QLineEdit" name="vol_feedback">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>601</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>故障信息</string>
    </property>
    <widget class="QPushButton" name="battery_low_vol_error">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>电池欠压故障</string>
     </property>
    </widget>
    <widget class="QPushButton" name="battery_over_vol_error">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>电池过压故障</string>
     </property>
    </widget>
    <widget class="QPushButton" name="motot1_commu_error">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>电机1通讯故障</string>
     </property>
    </widget>
    <widget class="QPushButton" name="motot2_commu_error">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>电机2通讯故障</string>
     </property>
    </widget>
    <widget class="QPushButton" name="motot3_commu_error">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>电机3通讯故障</string>
     </property>
    </widget>
    <widget class="QPushButton" name="motot4_commu_error">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>电机4通讯故障</string>
     </property>
    </widget>
    <widget class="QPushButton" name="motor_over_curr_error">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>141</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>电机驱动过温故障</string>
     </property>
    </widget>
    <widget class="QPushButton" name="motor_over_temp_error">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>60</y>
       <width>141</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>电机过流故障</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>931</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>运动控制反馈</string>
   </property>
   <widget class="QGroupBox" name="groupBox_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>移动车速</string>
    </property>
    <widget class="QLineEdit" name="line_speed_feedback">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_12">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>旋转速度</string>
    </property>
    <widget class="QLineEdit" name="angle_speed_feedback">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_13">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>键盘控制使能</string>
   </property>
   <widget class="QPushButton" name="key_mouse_enable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>使能</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_14">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>941</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>端口号：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="PortNumber_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>无</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>波特率：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="baudBox_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>数据位：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="DataBitsBox_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>停止位：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="StopBitsBox_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>56</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>校验位：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ParityBox_3">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>30</y>
      <width>77</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>无</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>偶校验</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>奇校验</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="StartOrStopButton_3">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>30</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(85, 170, 255)</string>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="key_note">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
